### PR TITLE
ESLint-ed All Backend Files

### PR DESCRIPTION
**Pull request by hongyi-chen**
Changes:

- Went through all of the files that ESLint applies on and linted them (and fixed a few syntactic bugs)
- Added a few `eslint-disable-next-line no-console`
- Added apollo-server-express package since some of the schemas needed it (not sure if we will use it later though)

I apologize in advance to whoever has to read through all these changes 😅

### CODE REVIEW

File: src/schemas/helloWorld.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-const { gql } = require('apollo-server')
+const { gql } = require('apollo-server');
```
Comment (hongyi-chen) [inline src/schemas/helloWorld.js]: I noticed that sometimes we're using `gql` from `apollo-server` and other times we're importing it from `apollo-server-express`, not sure if we want to change it for consistency so we only use one package instead

Comment (jzxhuang) [reply src/schemas/helloWorld.js]: I'm not sure what the difference is, but it's probably a good idea to import all from the same place (it seem like `apollo-server-express` would be the one) if it's all doing the same thing functionally

I skimmed through https://www.apollographql.com/docs/apollo-server/migration-two-dot/ but it's not all that clear to me lol

**Review by jzxhuang — APPROVED** [review]
Thanks for doing this! if you wanted to fix the `gql` thing go for it, otherwise this lgtm and I'll merge it in later in the week

File: package.json [hunk 2]
```diff
@@ -18,38 +18,30 @@
   },
   "homepage": "https://github.com/uwblueprint/building-up-server#readme",
   "devDependencies": {
-    "@types/node": "^14.10.1",
```
Comment (patrick-du) [inline package.json:21]: I removed all unused dependencies but someone should double-check to ensure I didn't remove anything important

Comment (hongyi-chen) [reply package.json:21]: @jzxhuang do you think we'll be using any of these? For instance `@types/cors`? I'm still not entirely sure what all of these packages are used for at the moment

Comment (jzxhuang) [reply package.json:21]: Any `@types/**` gives type declarations for Typescript

There's no need to include them since we're not using Typescript

**Review by patrick-du — APPROVED** [review]

**Pull request merged**